### ISSUE TITLE
TS2322 when checking vue pug templates

### ISSUE THREAD

**Issue by TSmulkys** [johnsoncodehk/vue-tsc#14]
Hi.
Then i use `href` or `src` attributes with pug template `TS2322` error appears (with html templates works fine)
Error appears only with `vue-tsc` `0.0.21` and above

#### Template example
```vue
<template lang="pug">
div
  a(href="#") Link text
  //- Or
  img(src="path/to/asset")
</template>
```

#### Error after run `vue-tsc --noEmit`
```
 error TS2322: Type 'true' is not assignable to type 'string | undefined'.
```

**Comment by johnsoncodehk** [comment]
Will check this, thanks for your report!

**Comment by johnsoncodehk** [comment]
Fixed in 0.0.23.
